infrastructure/kzg/src/test/java/tech/pegasys/teku/kzg/RustKZGTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright Consensys Software Inc., 2022
+ * Copyright Consensys Software Inc., 2024
```

Comment:
weird that its 2024...

Reply:
it was really first written in 2024
I've updated copyright years in all affected files

Reply:
ah makes sense